#binseg
reflectivity_threshold: 2000

#averager
#timeout_duration: 30.0
#enable_binaryseg: false #binary segmentation

#cluster
cluster_threshold: 10 # Minimum points for a main cluster density
#enable_averager: true # Average /scan topic over time
DBSCAN_epsilon : 0.005 # Maximum distance between two points for them to be considered as part of same cluster
DBSCAN_minpts : 1 # Minimum number of points required to form a dense region
reflector_radius : 0.0845 # Fixed Radius of the cylindrical reflector
#Reflector POSE
#radius: 